contrib/go/src/python/pants/contrib/go/subsystems/go_distribution.py

```diff
@@ -89,14 +61,14 @@ class GoCommand(namedtuple('GoCommand', ['cmdline', 'env'])):
     """Encapsulates a go command that can be executed."""
 
     @classmethod
-    def _create(cls, goroot, cmd, go_env, args=None):
+    def create(cls, goroot, cmd, go_env, args=None):
```

Comment:
Not a big objection, but it's not used publically fwict - perhaps you have plans?

Reply:
No plans, I think my IDE didn't like accessing a private member of another class.  But I guess it's fine if the class is a member of the same class as the code is in. I changed it back.